modules/private-clusters-setting-api-private.adoc

```diff
@@ -35,15 +35,19 @@ you can reconfigure the API server to use only the private zone.
 
 .. Locate and delete the appropriate load balancer component:
 ifndef::cpmso-using-azure[]
-*** For AWS, delete the external load balancer. The API DNS entry in the private zone already points to the internal load balancer, which uses an identical configuration, so you do not need to modify the internal load balancer.
+*** For {aws-short}, delete the external load balancer. The API DNS entry in the private zone already points to the internal load balancer, which uses an identical configuration, so you do not need to modify the internal load balancer.
 endif::cpmso-using-azure[]
 ifndef::cpmso-using-aws[]
-*** For Azure, delete the `api-internal-v4` rule for the public load balancer.
+*** For {azure-short}, you must delete the following resources:
+ ** The `api-v4` rule for the public load balancer.
+ ** The `frontenIPConfiguration` that is associated with the public load balancer.
```

Comment:
Here `frontenIPConfiguration` is associated with `api-v4` rule in public load balancer